packages/user-group-service/src/users/schema.ts

```diff
@@ -1,15 +1,30 @@
 import { Document, Model, model, Schema } from 'mongoose';
 
 export const UserSchema: Schema = new Schema({
-  name: String,
-  title: String,
+  cn: String,
```

Comment:
Can you create a alias `name` which returns the `cn` to make sure we don't break any other dependent services which are using the `name` field.